### ISSUE TITLE
question about the methods for pretraining entity/relation embeddings

### ISSUE THREAD

**Issue by yhjiujiu** [Hunter-DDM/InvTransE-and-InvRotatE#1]
Hi, thanks for your excellent work. However, there are a question: Does this work only need to train the pre-train model. Because the vector of the ookb entity is obtained  through the statistical information and pre-trained embeddings of in-kg entities. For example, can we use SACN or CompGCN to train the embedding of entity/relation? Thanks!

**Comment by Hunter-DDM** [comment]
Yes, this method can work only if we have pretrained TransE or RotatE embeddings. However, for other KGE methods, the usability of our method remains an open question. It can work if and only if we can calculate the only representation of an OOKG entity based on its neighbors. In addition, we think using other KGE methods as initialization and then training a TransE-like model can help you use our method.
